### DISCUSSION TITLE
Testing the app without success

### DISCUSSION THREAD

**Discussion by fscheps** [localsend/localsend#889] [Discussion (English, International)]
Hi community, today I found this app and I find it great that an alternative to AirDrop is being worked upon.
At our house we have a mix of devices between Mac´s and Windows PC´s. 
Today I needed to transfer a file from my Windows Desktop to my sons Windows Laptop, but I am getting the following error:

DioException [connection error]: The connection errored: The semaphore timeout period has expired.

Error: SocketException: The semaphore timeout period has expired.
 (OS Error: The semaphore timeout period has expired.
, errno = 121), address = 192.168.168.180, port = 60922

I am guessing its a problem of trusted relationship on firewalls? If so, it would be great if the installer could take care of whitelisting the app on the firewall of both machines.

I am trying to find the installation location of LocalSend, but I couldnt find any localsend folders on C:\Program Files nor on x86 folder. So not sure how to look for the app to manually whitelist it on both Windows firewalls. 

Maybe I am missing something, thanks a lot for your help ;)

**Comment by fscheps** [comment]
I had an idea to download the Portable version instead to check if this would allow me to add a Firewall exception, it did. But when I tried to execute the file I got: "The code execution cannot proceed because MSVCP140.DLL was not found. Reinstalling the program may fix this problem".

Not sure if this error message happened because before executing the portable version I uninstalled the previously MSIX installation. I am not very familiar with MSIX, but doesnt seemt o be behaving the same way a typical app would work, creating an installation folder on Program Files. 

Hope this ads a bit more clarity to be able to find a solution or enhancement. Thank you

**Reply by Tienisto** [reply]
You can checkout https://github.com/localsend/localsend/issues/682#issuecomment-1788360632

"Microsoft Visual C++2015-2022" would be likely needed in your case

**Comment by fscheps** [comment]
OK, so I found on the "More versions" link that there is an EXE package now, that would create a Folder on Program Files. So I downloaded it, installed it. When trying to execute it I receive multiple DLL errors:
You can see the errors here: https://photos.app.goo.gl/34rGEpivmdem7f6a9

Now I am lost, dont know how to clean this and reinstall the EXE to make it work. Thanks!
